br/tests/lightning_extend_routes/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Basic check for whether routing rules work
+
+set -eux
+
+for BACKEND in tidb local; do
+  run_sql 'DROP DATABASE IF EXISTS routes_a0;'
+  run_sql 'DROP DATABASE IF EXISTS routes_a1;'
+  run_sql 'DROP DATABASE IF EXISTS routes_b;'
+
+  run_sql 'CREATE DATABASE routes_b;'
+  run_sql 'CREATE TABLE routes_b.u (x real primary key, c_source varchar(11) not null, c_schema varchar(11) not null, c_table varchar(11) not null);'
```

Comment:
Could you add a case for ignore columns?

Reply:
addressed in d52d9e208